x-pack/docs/en/security/authentication/security-domain.asciidoc

```diff
@@ -22,7 +22,8 @@ Some types of resources in {es} are owned by a single user, such as
 <<async-search,async search contexts>>, <<security-api-create-api-key,API keys>>,
 and <<user-profile,user profiles>>. When a user creates a resource, {es}
 captures the user's username and realm information as part of the resource's
-metadata.
+metadata. Likewise, if a user updates a resource, such as an API key,
```

Comment:
Not strictly necessary but feels like a worthwhile callout. It makes the transition to the next paragraph slightly more clunky so I can also remove it/push it somewhere else.